keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -128,7 +128,9 @@ def __init__(self):
         # `tf.lookup.experimental.MutableHashTable` does not support string to
         # string mapping. So we first convert to string to an integer key, and
         # use the integer key to find the value.
-        self.factors = tf.pow(256, tf.range(0, 8, dtype=tf.int64))
+        self.factors = tf.pow(
+            tf.constant([256], dtype=tf.int64), tf.range(0, 8, dtype=tf.int64)
```

Comment:
Why do we need this in a list instead of just `tf.constant(256, dtype=tf.int64)`? I don't get why we need to change the shape.